modules/vaos/app/controllers/vaos/v1/base_controller.rb

```diff
@@ -18,7 +18,11 @@ class BaseController < VAOS::V0::BaseController
       private
 
       def fhir_service
-        VAOS::V1::FHIRService.new(current_user)
+        VAOS::V1::FHIRService.new(current_user, controller_name.singularize.capitalize.to_sym)
+      end
+
+      def query_string
+        request.fullpath.split('?').last
```

Comment:
Would the query string ever be blank?

Reply:
Good call, so far getting a list of resources is always a query but that could change.

Reply:
# File actionpack/lib/action_controller/request.rb, line 333
```ruby
    def query_string
      @env['QUERY_STRING'].present? ? @env['QUERY_STRING'] : (@env['REQUEST_URI'].split('?', 2)[1] || '')
    end
```

Reply:
Awesome 👍 Could you implement the edge case here and in the tests as well?